internal/pkg/install/install_unix.go

```diff
@@ -6,7 +6,7 @@ package install
 const DefaultRootParentDir string = "/opt"
 
 // DefaultSettingsDir - default dir for settings.
-const DefaultSettingsDir string = "/var/opt/ci-utils"
+const DefaultSettingsDir string = "/var/opt/ci-settings"
```

Comment:
This is going to require a change in the dockerfiles for `release-engineering` will it not? We put the `vault-util.toml` settings in this directory.

Reply:
Yep, I'll post those PRs for review.